apps/console/src/features/connections/pages/connections.tsx

```diff
@@ -107,7 +107,7 @@ const ConnectionsPage: FC<ConnectionsPropsInterface> = (props: ConnectionsPropsI
     const [ localAuthenticatorList, setLocalAuthenticatorList ] = useState<AuthenticatorInterface[]>([]);
     const [ filter, setFilter ] = useState<string>(null);
     const [ appendConnections, setAppendConnections ] = useState<boolean>(false);
-    const [ isPaginating, setIsPaginating ] = useState<boolean>(false);
+    const [ isPaginating ] = useState<boolean>(false);
```

Comment:
If we are not setting this state variable, we can remove it

Reply:
Resolved via https://github.com/wso2/identity-apps/pull/4460/commits/87e1442cda8f268962da73384711efa2a5daa023